src/cli.rs

```diff
@@ -92,6 +92,11 @@ pub enum Cmd {
         #[command(subcommand)]
         command: ConfigCmd,
     },
+    #[command(
+        name = "completion",
+        about = "outputs a completion script for a given shell"
+    )]
```

Comment:
First letter should be capitalize. Also, let's stick with description through comment to keep the file consistent. Change "outputs" to "print",  to also keep consistency.
```suggestion
    // Print completion script for a given shell
    #[command(name = "completion")]
```